src/main/java/menion/android/whereyougo/geo/location/Location.java

```diff
@@ -7,10 +7,10 @@ public Location() {
     }
 
     public Location(android.location.Location loc) {
-        this(loc.getProvider(), loc.getLatitude(), loc.getLongitude());
+        this(loc.getLatitude(), loc.getLongitude());
         setTime(loc.getTime());
         if (loc.hasAccuracy()) {
-            setAccuracy(loc.getAccuracy());
+            setAccuracyHor(loc.getAccuracy());
         }
```

Comment:
Please add the part vor the vertical Accurency:
~~~
        if (loc.hasVerticalAccuracy()) {
            setAccuracyVer(loc.getVerticalAccuracyMeters());
        }
~~~